app/Policies/IndividualPolicy.php

```diff
@@ -63,9 +63,13 @@ public function update(User $user, Individual $individual): Response
 
     public function publish(User $user, Individual $individual): Response
     {
-        return $user->id === $individual->user_id && $individual->isPublishable()
+        if ($user->id !== $individual->user_id) {
+            return Response::deny(__('You cannot publish this individual page.'));
+        }
+
+        return $individual->isPublishable()
             ? Response::allow()
-            : Response::deny(__('You cannot publish this individual page.'));
+            : Response::deny(__('This individual page is missing some required information, and cannot be published yet.'));
```

Comment:
It doesn't appear that it is actually possible to hit the first condition. I've tried to write tests for it, but it gets caught on other policies first. I'm planning to revert this change to keep the more generic message just in case something does slip through.